apps/static/dance/ai/model/cachedbackgroundmap.json

```diff
@@ -0,0 +1,192 @@
+{
+  "emojiAssociations": {
+    "art": {
+      "association": [
+        3, 3, 2, 2, 3, 2, 2, 2, 2, 1, 2, 2, 2, 2, 2, 3, 2, 2, 2, 2, 2, 2, 2, 2
+      ],
+      "explanation": "The word 'art' is closely associated with 'circles,' 'colors,' and 'paint splatter' backgrounds, which often feature artistic and vibrant patterns."

```

Comment:
I suspect we will not be using these LLM explanations after all, in which case I think best to remove them for now, until we figure out exactly what we are going to do.

Reply:
Noted, I'll keep them for now in the cached file just in case but will generate the google emoji json without them